### PR TITLE
modified learn link

### PR DESCRIPTION
**Pull request by vinnieangel**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by vinnieangel** [comment]
#sign-off

**Comment by azure-quickstarts** [comment]
@aahill - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.cognitiveservices/cognitive-services-universalkey/metadata.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "level": "beginner",
     "cost": "$",
     "roles": "AI Engineer",
-    "learning": "https://docs.microsoft.com/en-us/learn/browse/?products=azure&term=cognitive&WT.mc_id=Portal-Microsoft_Azure_Education&terms=cognitive",
+    "learning": "https://learn.microsoft.com/en-us/training/browse/?products=azure&term=cognitive&WT.mc_id=portaledu_inproduct_template&terms=cognitive",
```
Comment (alex-frankel) [inline quickstarts/microsoft.cognitiveservices/cognitive-services-universalkey/metadata.json:17]: please remove en-us from link so locale can be selected automatically

**Review by alex-frankel — CHANGES_REQUESTED** [review]

**Comment by alex-frankel** [comment]
Closing due to no response

**Pull request closed without merging**